src/engine/level_script.c

```diff
@@ -31,7 +31,7 @@
 
 #include "config.h"
 
-#define NUM_PAINTINGS 45
+#define NUM_PAINTING_WARP_NODES 45
```

Comment:
`NUM_PAINTING_WARP_NODES` was removed since paintings use regular object warp nodes now, but I'll add documentation/instructions in paintings.h for setting up painting structs and how warps are triggered.